code/modules/projectiles/updated_projectiles/guns/smgs.dm

```diff
@@ -216,6 +216,9 @@
 	origin_tech = "combat=3;materials=2;syndicate=4"
 	fire_sound = 'sound/weapons/smg_heavy.ogg'
 	current_mag = /obj/item/ammo_magazine/smg/ppsh
+	attachable_allowed = list(
+						/obj/item/attachable/flashlight)
```

Comment:
You did test how this looks, right?

Reply:
![image](https://user-images.githubusercontent.com/25509881/44738210-8383df80-aaf4-11e8-82e8-febc3baf6087.png)